src/main.js

```diff
@@ -168,3 +173,38 @@ function returnToMainPage() {
   returnToMainPageButton.classList.add('hidden');
   savedPosterPage.classList.add('hidden');
 }
+
+ function createCustomPoster() {
+  // images.push(imageInput.value);
+  // titles.push(titleInput.value);
+  // quotes.push(quoteInput.value);
+  createNewPoster(imageInput.value,titleInput.value,quoteInput.value);
+  saveData(imageInput.value,titleInput.value,quoteInput.value);
+  returnToMainPage();
+  displayNewPoster(imageInput.value,titleInput.value,quoteInput.value);
+   console.log(true);
+}
+//
+function createNewPoster(imageURL,title,quote) {
+  var newCustomPoster = new Poster(imageURL, title, quote);
```

Comment:
The same is true here, no need to declare the global.  As well, Instantiating a new class does not require that we rename that class throughout the main.js file.